### ISSUE TITLE
Mutation-only server not supported

### ISSUE THREAD

**Issue by thejibz** [smallrye/smallrye-graphql#192]
When creating a server with only mutations, the following error appear:
```
{"WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"server.war\".undertow-deployment" => "java.lang.RuntimeException: graphql.AssertException: queryType can't be null
    Caused by: java.lang.RuntimeException: graphql.AssertException: queryType can't be null
    Caused by: graphql.AssertException: queryType can't be null"}}
```

Example of Java code to reproduce:

```
@GraphQLApi
public class TckGraphQLApi {

    @Mutation
    public String myMutation() {
        return "mutation";
    }
}
```

**Comment by thejibz** [comment]
You're welcome @phillip-kruger ! I use this:

```
<dependency>
            <groupId>io.smallrye</groupId>
            <artifactId>smallrye-graphql-servlet</artifactId>
            <version>1.0.1</version>
</dependency>
```

**Comment by phillip-kruger** [comment]
Ok Thanks. It might be fixed in 1.0.2-SNAPSHOT already. I'll double-check

**Comment by phillip-kruger** [comment]
This was indeed an issue even in master. It's now fixed and will be part of 1.0.2
